src/frontend/Ast_to_Mir.ml

```diff
@@ -584,8 +584,14 @@ let rec trans_stmt (declc : decl_context) (ts : Ast.typed_statement) =
       raise_s
         [%message
           "Found function definition statement outside of function block"]
+  | Ast.VarDecl {decl_type= Unsized _; _} ->
+      raise_s [%message "Don't yet support unsized declarations"]
```

Comment:
Why do we not yet support unsized declarations at this point? We should be able to translate them to the MIR, right? Isn't it the code generation that's problematic?

Reply:
Yeah, it could be supported it just isn't in this PR. I figured whoever first starts emitting it will have to change it 😂 (figuring it'd be me in the simplify-indexing branch).

Reply:
I'm working on adding it now tho... might as well.